ci/common/common.sh

```diff
@@ -147,3 +153,23 @@ commit_subtree() {(
     fi
   fi
 )}
+
+with_token() {(
+  set +x
+  if [[ $1 = --empty-unset ]] ; then
+    : ${GH_TOKEN:=}
+    shift
+  else
+    set -u
+    : ${GH_TOKEN}
+  fi
+  for arg ; do
+    arg="${arg//%token/$GH_TOKEN}"
+    arg="${arg//%%/%}"
+    printf '%s\0' $arg
+  done | xargs -0 -x sh -c '"$@"' -
```

Comment:
@ZyX-I What is the purpose of `-` here? I am seeing `-: -n: command not found` in some cases.